src/pages/ImportWorkflow/ImportWorkflow.js

```diff
@@ -85,9 +86,15 @@ export const ImportWorkflow = ({ path, version, source }) => {
         method_output_mappings: [],
       };
       const res = await Ajax(signal).Cbas.methods.post(appUrls.cbasUrl, postRequestBody);
+      const methodId = res.method_id;
 
       await setAzureCookieOnUrl(signal, appUrls.cbasUiUrl, true);
```

Comment:
TOL - Do we need to set the CBAS UI cookie if we're just redirecting within Terra?

Reply:
I don't think so. I had this line x2 and maybe I removed the wrong one. @salonishah11 can you confirm? Can I move this within the `else` block?

Reply:
Testing this out, it looks like we can set the cookie from within the `else` block, and I refactored as such. Thanks @dillydally414